README.md

```diff
@@ -221,54 +222,89 @@ catch (TerminalStateException exception)
 For more usage examples check out our [ASP.NET Sample Application](samples/samples-aspnet).
 
 ## Configuration Reference
-  
+
 This library looks for configuration in the following sources:
- 
-1. An `okta.yaml` file in a `.okta` folder in the current user's home directory (`~/.okta/okta.yaml` or `%userprofile%\.okta\okta.yaml`)
-2. An `okta.yaml` file in a `.okta` folder in the application or project's root directory
-3. Environment variables
-4. Configuration explicitly passed to the constructor (see the example in [Getting started](#getting-started))
- 
-Higher numbers win. In other words, configuration passed via the constructor will override configuration found in environment variables, which will override configuration in `okta.yaml` (if any), and so on.
- 
+
+1. As a YAML file in your home directory
+2. As part of an appsettings.json file in your application’s working directory
+3. As a YAML file in your application’s working directory
+4. In environment variables
+5. In the client constructor for the application  (see the example in [Getting started](#getting-started))
+
+This is the order that the locations are searched for configuration information. If information is set in more than one location, it is overwritten so if, for example, the Issuer is set in both an okta.yaml file and in environment variables, the value in the environment variables takes precedence. If the Issuer was also set in the client constructor, that value would be used instead of either of the other two.
+
+> Note: To avoid confusion as to which configuration values are used by the SDK, you should use only one configuration option in your solution.
+
+
 ### YAML configuration
- 
-The full YAML configuration looks like:
- 
-```yaml
+
+The SDK will look for a YAML file named `okta.yaml` in:
+
+- A folder called .okta whose parent is the current user's home directory
+  - On Mac/Linux, this is `~/.okta/okta.yaml`
+  - On Windows, this is `%userprofile%\.okta\okta.yaml`
+- The application's current working directory
+  - If you're using the recommended IISExpress debugger in Visual Studio to run your app, this is `${IIS Express install location}\IIS Express`, for example, `C:\Program Files\IIS Express\okta.yaml`.
+
+The information in the YAML file should be formatted as follows:
+
+```
 okta:
   idx:
-    issuer: "https://{yourOktaDomain}/oauth2/{authorizationServerId}" # e.g. https://foo.okta.com/oauth2/default, https://foo.okta.com/oauth2/ausar5vgt5TSDsfcJ0h7
-    clientId: "{clientId}"
-    clientSecret: "{clientSecret}" # Required for confidential clients
+    issuer: "https://${yourOktaDomain}/oauth2/default"
+    clientId: "${clientId}"
+    clientSecret: "${clientSecret}"
```

Comment:
Do we need these dollar signs? These properties now look somewhat like string interpolation patterns and this can be confusing.
May be it's better to use the same constants as in the lines 67-69: "{YOUR_ISSUER}", "{YOUR_CLIENT_ID}" and so on.

Reply:
I agree with Andrii here.